binder/environment.yml

```diff
@@ -0,0 +1,38 @@
+name: sunpy
+channels:
+  - conda-forge
+dependencies:
+  - python
+  - pip
+  - openjpeg
+  - jsonschema=2.6.0
+  - numpy
+  - scipy
+  - matplotlib
+  - pandas
+  - astropy
+  - parfive
+  - sqlalchemy
+  - scikit-image
+  - glymur
+  - beautifulsoup4
+  - drms
+  - python-dateutil
+  - zeep
+  - tqdm
+  - asdf
+  - hypothesis
+  - pytest-astropy
+  - pytest-cov
+  - pytest-mock
+  - ruamel.yaml
+  - sphinx
+  - sphinx-astropy
+  - sphinx-gallery
+  - towncrier
+  - pip:
+    - sunpy-sphinx-theme
+    - jplephem
+    - astroquery
+    - jupytext
+    - ..
```

Comment:
What is this?

Reply:
It installs the git repo in the directory above. Given I asked the [same question](https://github.com/sunpy/sunpy/pull/3256#discussion_r299040618) we should probably comment this.